main.js

```diff
@@ -642,6 +642,10 @@ const log = msg => {
   if (!msg) return;
   global.serverLog += msg;
 
+  // We wont let the server log get too long otherwise it could fill up all the ram
+  // and crash the client.
+  global.serverLog = global.serverLog.slice(global.serverLog.length - 1000000);
```

Comment:
This caps the log we keep in memory to 1MB and will keep the most recent part of it.